lib/solr/document.rb

```diff
@@ -12,15 +12,16 @@ def self.empty
       end
     end
 
-    attr_reader :id, :model_name, :score, :debug_info, :group
+    attr_reader :id, :model_name, :score, :debug_info, :group, :fields
 
     # TODO: model_name is specific for machinio
-    def initialize(id:, model_name: nil, score: nil, debug_info: nil, group: GroupInformation.empty)
+    def initialize(id:, model_name: nil, score: nil, debug_info: nil, group: GroupInformation.empty, fields: {})
       @id = id
       @model_name = model_name
       @score = score
       @debug_info = debug_info
       @group = group
+      @fields = fields
```

Comment:
Should it be `fields` or `attributes`? This value stores data fetched from solr specified in `response_fields` (like name, title, etc).

Reply:
`fields` sounds ok to me